qemu/qmp.go

```diff
@@ -805,6 +805,12 @@ func (q *QMP) ExecuteNetPCIDeviceAdd(ctx context.Context, netdevID, devID, macAd
 	if bus != "" {
 		args["bus"] = bus
 	}
+
+	if fdNum > 0 {
+		args["mq"] = "on"
+		args["vectors"] = 2*fdNum + 2
```

Comment:
please add some comments explaining this.  why 2 and not 3 or 4 ?

Reply:
These codes come from https://github.com/intel/govmm/blob/6aa35d33f2fded7ee32f7eadabda95182059572f/qemu/qemu.go#L497-L502